Junit/test/IPLTest.java

```diff
@@ -0,0 +1,12 @@
+import static org.junit.jupiter.api.Assertions.*;
+
+class IPLTest {
+
+    // here findBestBatsmenTeam function is tested.
+
+    @org.junit.jupiter.api.Test
+    void findBestBatsmenTeamTest() {
```

Comment:
I see that you have tried to add a test case to validate your program, but this is not the right way to do it. 
Get on a call with any of the track leads or assists to under the proper usage here.